perses/rjmc/geometry.py

```diff
@@ -193,11 +199,17 @@ def logp_reverse(self, top_proposal, new_coordinates, old_coordinates):
 
                 #propose a bond and calculate its probability
                 bond = self._get_relevant_bond(atom, bond_atom)
-                logp_r = self._bond_logp(internal_coordinates[0], bond, top_proposal.beta)
+                bond_k = bond.type.k*units.kilocalorie_per_mole/units.angstrom**2
+                sigma_r = units.sqrt(1/(beta*bond_k))
+                logZ_r = np.log((np.sqrt(2*np.pi)*sigma_r/sigma_r.unit)) #need to eliminate unit to allow numpy log
+                logp_r = self._bond_logq(internal_coordinates[0], bond, beta) - logZ_r
 
                 #propose an angle and calculate its probability
                 angle = self._get_relevant_angle(atom, bond_atom, angle_atom)
-                logp_theta = self._angle_logp(internal_coordinates[1], angle, top_proposal.beta)
+                angle_k = angle.type.k*units.kilocalorie_per_mole/(units.degrees**2)
```

Comment:
As below, this is not a conversion. It's just assigning the correct unit given the `ParmEd` defaults...unless I'm mistaken about those.